pkgs/applications/editors/emacs/build-support/generic.nix

```diff
@@ -21,21 +21,26 @@
     fi
   '';
 
+  libBuildHelper = import ./lib-build-helper.nix { inherit lib; };
+
 in
 
+libBuildHelper.adaptMkDerivation { } stdenv.mkDerivation (finalAttrs:
+
 { pname
```

Comment:
definition `version` is not used

Reply:
indirectly used as `finalAttrs.version` here https://github.com/NixOS/nixpkgs/pull/330589/files#diff-4d0cbd8553ca2ab2a1daac35346bf2ae8322d3d69cf5dc64f1413e84a784a13bR43